### DISCUSSION TITLE
How to download

### DISCUSSION THREAD

**Discussion by charlesf218** [Zodsmar/SeedSearcherStandaloneTool#82] [Q&A] [answered]
Look, I really want to use this. I don't know how to download it. Like i downloaded the zip file but where is the main menu? Like where is the place where you insert your requirements. I launched the gradlew.bat and it comes up with the console and all but I am wondering where  I can get to the place I have seen in videos. Can someone please help?

**Comment by Red3Tango** [comment, marked as answer]
Grab the .exe or .jar file from the releases page. You are currently downloading the source files.

Code -> Releases (right side) -> Assets (bottom of each release)

**Reply by charlesf218** [reply]
Alright i got it all worked out now thank you so much!
